src/query/config/src/inner.rs

```diff
@@ -289,11 +291,17 @@ impl Default for MetaConfig {
             auto_sync_interval: 10,
             rpc_tls_meta_server_root_ca_cert: "".to_string(),
             rpc_tls_meta_service_domain_name: "localhost".to_string(),
+            meta_type: serde_json::to_string(&MetaType::Embedded).unwrap(),
```

Comment:
Don't use `serde_json::to_string`.

![image](https://user-images.githubusercontent.com/5351546/199434990-bea37ac6-547b-4041-8605-2412ec6a2907.png)

Reply:
> Don't use `serde_json::to_string`.

Any suggestions about what to use? 🤔

Reply:
Use `MetaType` directly in `InnerMetaConfig` and `String` in `OuterMetaConfig`.

Reply:
I use `strum_macros` in `MetaType` to convert to\from string.